adapters/rest-vertx/src/main/java/org/eclipse/hono/adapter/rest/VertxBasedRestProtocolAdapter.java

```diff
@@ -62,4 +83,23 @@ private static String getDeviceIdParam(final RoutingContext ctx) {
     protected String getStatusResourcePath() {
         return null;
     }
+
+    @Override
+    public final void authenticate(JsonObject authInfo, Handler<AsyncResult<User>> resultHandler) {
```

Comment:
can you please declare the params to be final

Reply:
Yup all done

Reply:
No, not the methods, just the params, i.e. 
`public final void authenticate(final JsonObject authInfo, final Handler....)`

There is no need to declare private methods final, they cannot be overridden anyway.